ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -46,6 +46,14 @@
 #include <tensorflow/contrib/lite/kernels/register.h>
 #endif
 
+#if defined(__ANDROID__) && (TFLITE_VERSION_MAJOR >= 2 || TFLITE_VERSION_MINOR >= 14)
+#include <tensorflow/lite/delegates/nnapi/nnapi_delegate.h>
+#endif
+
+#if defined(__ANDROID__) && (TFLITE_VERSION_MAJOR >= 2 && TFLITE_VERSION_MINOR >= 3)
```

Comment:
Does this mean that TFLite 3.0 will not support gpu delegate? :)

Reply:
Would more specific check: `(TFLITE_VERSION_MAJOR == 2 && TFLITE_VERSIONM_MINOR >=3)` be reasonable?

Reply:
```((TFLITE_VERSION_MAJOR == 2 && TFLITE_VERSIONM_MINOR >=3) || TFLITE_VERSION_MAJOR > 2)```

Reply:
Updated, Please check!